### DISCUSSION TITLE
How to remove deleted repository content since it is appearing in Turnitin submission<title>

### DISCUSSION THREAD

**Discussion by resmyvijay** [community/community#56690] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, I am a Master's student at Universiti Malaya. I am trying to publish a research paper based on my research project. I had created my project as a repository on github. I had used my project report content in the Readme.md file. When I try to check my research paper for plagiarism using turnitin, it shows my own github repository content as source for plagiarism. I then deleted my repository on github. It has been two weeks now, but the cached repository content still shows up in turnitin as source of plagiarism for my research paper. When I click on the link in turnitin to my repository, it shows 404 Not Found. How can I completely erase the repository so that it is not detected by turnitin even from the cache?
The deleted repository is https://github.com/resmyvijay/Replacing-Furniture-Image-From-Indoor-Scenes-Using-Generative-Adversarial-Networks-GAN

Thanks in advance for your help.

**Comment by Itz0xAkira** [comment]
Hello :D,
so here is the thing... when you delete a repo on github, it stays in the deleted repos for 90 days

![image](https://github.com/community/community/assets/14275784/e76bdde1-d4f4-4e29-848d-404d760ff849)

you can check it here [https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#about-repository-restoration](url), so after 90 days it will be "permanently" deleted. I think that is what causing the issue. your best bet is to just restore it and empty the Readme.md file by yourself that might fix the issue.

**Reply by airtower-luna** [reply]
I don't think GitHub keeping the repository for restoration is the problem, because it won't be public that way.

More likely Turnitin simply keeps once collected data in their database (which sort of makes sense for the purpose). Publishing something you wrote yourself in two different places is not plagiarism (which is defined as copying from someone else) as such. However I can't say if *having published the same thing before* might get you in trouble in your specific situation, I wouldn't be too surprised if there's some rule about newly published papers needing to be actually new. That might be something to discuss with your academic advisor.

**Reply by Itz0xAkira** [reply]
yes this is actually a better explanation, thankQ @airtower-luna  :D
